SDVModTest/UIElements/ExperienceBar.cs

```diff
@@ -107,20 +153,20 @@ public void ToggleShowExperienceBar(bool showExperienceBar)
 
         private void OnLevelUp(object sender, EventArgsLevelUp e)
         {
-            if (_showLevelUpAnimation)
+            if (_showLevelUpAnimation.IsOn)
             {
                 switch (e.Type)
                 {
-                    case EventArgsLevelUp.LevelType.Combat: _levelUpIconRectangle.X = 120; break;
-                    case EventArgsLevelUp.LevelType.Farming: _levelUpIconRectangle.X = 10; break;
-                    case EventArgsLevelUp.LevelType.Fishing: _levelUpIconRectangle.X = 20; break;
-                    case EventArgsLevelUp.LevelType.Foraging: _levelUpIconRectangle.X = 60; break;
-                    case EventArgsLevelUp.LevelType.Mining: _levelUpIconRectangle.X = 30; break;
+                case EventArgsLevelUp.LevelType.Combat: _levelUpIconRectangle.X = 120; break;

```

Comment:
case statements are inside a new scope in the switch statement. They should be tabbed in.